[2025-01-17T20:19:49.129+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T20:19:49.147+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spaceflight_data_ingest_v2.fetch_articles manual__2025-01-17T19:59:04.875057+00:00 [queued]>
[2025-01-17T20:19:49.155+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spaceflight_data_ingest_v2.fetch_articles manual__2025-01-17T19:59:04.875057+00:00 [queued]>
[2025-01-17T20:19:49.155+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-17T20:19:49.170+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_articles> on 2025-01-17 19:59:04.875057+00:00
[2025-01-17T20:19:49.179+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T20:19:49.181+0000] {standard_task_runner.py:72} INFO - Started process 210 to run task
[2025-01-17T20:19:49.181+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spaceflight_data_ingest_v2', 'fetch_articles', 'manual__2025-01-17T19:59:04.875057+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/spaceflight_data_ingest_v2.py', '--cfg-path', '/tmp/tmp1my3lg3g']
[2025-01-17T20:19:49.182+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask fetch_articles
[2025-01-17T20:19:49.226+0000] {task_command.py:467} INFO - Running <TaskInstance: spaceflight_data_ingest_v2.fetch_articles manual__2025-01-17T19:59:04.875057+00:00 [running]> on host 59ff16f1d4fb
[2025-01-17T20:19:49.527+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spaceflight_data_ingest_v2' AIRFLOW_CTX_TASK_ID='fetch_articles' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T19:59:04.875057+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T19:59:04.875057+00:00'
[2025-01-17T20:19:49.528+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-17T20:19:49.528+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-17T20:19:49.529+0000] {logging_mixin.py:190} INFO - Current task name:fetch_articles state:running start_date:2025-01-17 20:19:49.147637+00:00
[2025-01-17T20:19:49.529+0000] {logging_mixin.py:190} INFO - Dag name:spaceflight_data_ingest_v2 and current dag run status:running
[2025-01-17T20:19:49.529+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T20:19:50.091+0000] {spaceflight_data_ingest_v2.py:44} INFO - No hay más artículos disponibles.
[2025-01-17T20:19:50.092+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T20:19:50.102+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T20:19:50.103+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=spaceflight_data_ingest_v2, task_id=fetch_articles, run_id=manual__2025-01-17T19:59:04.875057+00:00, execution_date=20250117T195904, start_date=20250117T201949, end_date=20250117T201950
[2025-01-17T20:19:50.116+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-17T20:19:50.117+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-17T20:19:50.117+0000] {logging_mixin.py:190} INFO - Dag name:spaceflight_data_ingest_v2 queued_at:2025-01-17 19:59:04.907203+00:00
[2025-01-17T20:19:50.117+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-17T20:19:50.170+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T20:19:50.384+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-17T20:19:50.388+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-19T04:07:33.874+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-19T04:07:33.945+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: spaceflight_data_ingest_v2.fetch_articles manual__2025-01-17T19:59:04.875057+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2025-01-19T04:07:33.962+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
